charts/spire/charts/spiffe-oidc-discovery-provider/templates/roles-test.yaml

```diff
@@ -0,0 +1,38 @@
+# Role to be able to manage cert requests with Cert-Manager
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can the extra roles/bindings/sa/etc be annotated tests too so they only get created when needed?

Reply:
done